approval_frame/forms.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from registration.forms import RegistrationFormUniqueEmail
+
+class RegistrationFormCustom(RegistrationFormUniqueEmail):
+    """
+    Subclass of `RegistrationFormUniqueEmail` which includes application-
```

Comment:
Hmm yes, I totally get your point.  Nice article! 
Though I was trying to mimic the documentation style of the classes here in [django-registration forms module] (https://github.com/macropin/django-registration/blob/master/registration/forms.py) . Perhaps it was done that way for the Sphinx documentation ? I have modified the comment though